src/css/style.scss

```
@@ -780,3 +996,136 @@ button:focus-visible, a:focus-visible, input:focus-visible, textarea:focus-visib
+		line-height: 1.6;
+	}
+
+	.wppo-dialog-actions {
+		display: flex;
+		justify-content: flex-end;
+		gap: 12px;
+	}
+
+	.wppo-dialog-detail-list {
+		list-style: none;
+		padding: 0;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)